src/routes/router.ts

```diff
@@ -54,6 +56,10 @@ RootRouter.use(MaintenanceRouter);
 RootRouter.use("/github", GithubRouter);
 RootRouter.use("/jira", JiraRouter);
 
+RootRouter.post("/webhook/event/:uuid",
+	header(["x-github-event", "x-hub-signature-256", "x-github-delivery"]).not().isEmpty().withMessage("Missing Header"),
```

Comment:
I think the convent we use is would be something like `header(["x-github-event", "x-hub-signature-256", "x-github-delivery"]).exists(), returnOnValidationError`